app/serializers/concerns/user_sidebar_mixin.rb

```diff
@@ -16,33 +16,10 @@ def include_display_sidebar_tags?
   end
 
   def include_sidebar_tags?
-    SiteSetting.tagging_enabled && sidebar_navigation_menu?
+    SiteSetting.tagging_enabled
   end
 
   def sidebar_category_ids
     object.secured_sidebar_category_ids(scope)
   end
-
-  def include_sidebar_category_ids?
-    sidebar_navigation_menu?
-  end
-
-  def sidebar_sections
-    object
-      .sidebar_sections
-      .order(created_at: :asc)
-      .includes(sidebar_section_links: :linkable)
-      .map { |section| SidebarSectionSerializer.new(section, root: false) }
-  end
```

Comment:
This has been moved to `UserSerializer` instead of being in the mixin which is included in `CurrentUserSerializer` which has its own override of `sidebar_sections`.